docs/markdown/i18n-module.md

```diff
@@ -74,3 +74,72 @@ for normal keywords. In addition it accepts these keywords:
 * `mo_targets` *required*: mo file generation targets as returned by `i18n.gettext()`.
 
 *Added 0.62.0*
+
+
+### i18n.pot_extractor()
```

Comment:
I'd prefer a more specific name like `find_xgettext` for this function and `XgettextProgram` for the object.

Reply:
While I'm open changing the names, I'm not sure about `find_xgettext`, because I think it would suggest that the returned object is compatible with was is returned by `find_program`. Or should it be?

Reply:
Actually, I think this is a neat idea. I modified it to make the returned object a subclass of ExternalProgram.

Reply:
Finally refactored as a simple function, so it no longer applies.